configurator/src/config.template

```diff
@@ -4,6 +4,12 @@ bitcoin-rpcpassword={bitcoin_rpc_pass}
 bitcoin-rpcconnect={bitcoin_rpc_host}
 bitcoin-rpcport={bitcoin_rpc_port}
 plugin=/usr/local/libexec/c-lightning/plugins/c-lightning-http-plugin
+plugin=/usr/local/libexec/c-lightning/plugins/rebalance/rebalance.py
+plugin=/usr/local/libexec/c-lightning/plugins/summary/summary.py
+plugin=/usr/local/libexec/c-lightning/plugins/c-lightning-REST/plugin.js
```

Comment:
does this mean they are all turned on by default?

Reply:
Done.